emcc.py

```diff
@@ -1195,20 +1192,14 @@ def get_last_setting_change(setting):
 
     shared.verify_settings()
 
-    if final_suffix in WASM_ENDINGS and shared.Settings.WASM_BACKEND and not shared.Settings.SIDE_MODULE:
+    if final_suffix in WASM_ENDINGS and not shared.Settings.SIDE_MODULE:
       # if the output is just a wasm file, it will normally be a standalone one,
       # as there is no JS. an exception are side modules, as we can't tell at
       # compile time whether JS will be involved or not - the main module may
       # have JS, and the side module is expected to link against that.
       # we also do not support standalone mode in fastcomp.
       shared.Settings.STANDALONE_WASM = 1
 
-    # We allow this error to be supressed by the environment so that we can run the test
-    # suite against fastcomp for the time being.
-    # See: https://github.com/emscripten-core/emscripten/issues/11319
-    if not shared.Settings.WASM_BACKEND and 'EMCC_ALLOW_FASTCOMP' not in os.environ:
```

Comment:
We still want this right?

Reply:
Hmm, why would we want to look at `EMCC_ALLOW_FASTCOMP`? Fastcomp won't work with all the removed code anyhow, and we aren't running those tests.

Or do you mean we want an error for trying to use fastcomp? I don't think it's possible to get to this code location. We error anyhow on trying to set `WASM_BACKEND` - we always have, before this PR, it's autodetected. And we've even removed that autodetection code?

Reply:
I guess I mean we still need the error case... but you can drop the EMCC_ALLOW_FASTCOMP part